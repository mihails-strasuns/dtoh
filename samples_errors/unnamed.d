module unnamed;
extern(C) int foo ( void function() );
