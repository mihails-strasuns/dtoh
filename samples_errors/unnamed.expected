samples_errors/unnamed.d(2) [extern (C) void()]
	C does not support unnamed function pointers, consider using alias or named parameter
